### ISSUE TITLE
initial vector position does not snap to grid

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/vector-addition#213]
Reported by @kathy-phet via email:

>  I found this which I think might be a small bug. Pull out two vectors then put tip to tail. Click show sum vector - it's tail doesn't start on a grid line, but it seems like it should. 

![pchbbpenilnhpkeg](https://user-images.githubusercontent.com/3046552/66097260-e26b2200-e55a-11e9-97b1-002c1a682077.png)

**Comment by pixelzoom** [comment]
This appears to be the culprit, in `LabGraph`:

```js
      // @public (read-only) {VectorSet} vectorSet1
      this.vectorSet1 = new VectorSet( this, componentStyleProperty, sumVisibleProperty1, vectorColorPalette1, {
        initialSumTailPosition: new Vector2( LAB_GRAPH_BOUNDS.minX + 1 * LAB_GRAPH_BOUNDS.width / 3, LAB_GRAPH_BOUNDS.centerY )
      } );

      // @public (read-only) {VectorSet} vectorSet2
      this.vectorSet2 = new VectorSet( this, componentStyleProperty, sumVisibleProperty2, vectorColorPalette2, {
        initialSumTailPosition: new Vector2( LAB_GRAPH_BOUNDS.minX + 2 * LAB_GRAPH_BOUNDS.width / 3, LAB_GRAPH_BOUNDS.centerY )
      } );
```

Those `initialSumTailPosition` values are `{x: 11.666666666666668, y: 10}`  and `{x: 28.333333333333336, y: 10}` respectively.

**Comment by pixelzoom** [comment]
Fixed in master and [1.0.0-dev.35](https://phet-dev.colorado.edu/html/vector-addition/1.0.0-dev.35/phet/vector-addition_en_phet.html).   @arouinfar FYI.

Closing.
